modules/nf-core/multiqc/tests/main.nf.test

```diff
@@ -26,20 +23,17 @@ nextflow_process {
         then {
             assertAll(
                 { assert process.success },
-                { assert path(process.out.report.get(0)).exists() },
-                { assert path(process.out.data.get(0)).exists() },
-                { assert path(process.out.versions.get(0)).getText().contains("multiqc") }
+                { assert process.out.report[0] ==~ ".*/multiqc_report.html" },
+                { assert process.out.data[0] ==~ ".*/multiqc_data" },
```

Comment:
Is the output data also unstable? The html report I understand, but just wandering about the rest as well.